### ISSUE TITLE
Calendar Header Styling

### ISSUE THREAD

**Issue by srinig29** [rebeccaXam/XamForms.Controls.Calendar#8]
Hi, 
Is it possible to change the calendar header style. We tried, but couldn't change only the header style, rest all other background, textcolor etc.. can be changed.

**Comment by rebeccaXam** [comment]
Yes, you can change some styling of the header. The left/right navigation buttons and the title label are exposed by ```TitleLabel```, ```TitleLeftArrow```, ```TitleRightArrow``` and ```MonthNavigationLayout```. The ```MonthNavigationLayout``` is the stack layout the label and buttons are embedded inside.

So changing some styling of the header would be:
```c#
calendar.TitleLeftArrow.TextColor = Color.Blue; // Set the text color of the left arrow to blue
calendar.TitleRightArrow.TextColor = Color.Blue; // Set the text color of the right arrow to blue
calendar.TitleLabel.TextColor = Color.Blue; // Set the text color of the header text to blue
calendar.TitleLabelFormat = "MM-yyyy"; // Set a different date format of the title label
calendar.MonthNavigationLayout.HeightRequest = 75; // Set the height of the header to 75
calendar.MonthNavigationLayout.BackgroundColor = Color.Green; // Set the background color of the header to green
```

If you really want to change the header and the above options are not enough, please hide the header by  ```MonthNavigationShow= false``` and you can then place a custom header above the calendar. For custom month navigation add:
```c#
// Add this to an event, that will trigger the calendar to go 1 month back
calendar.StartDate = new DateTime(StartDate.Year, StartDate.Month, 1).AddMonths(-1);

// Add this to an event, that will trigger the calendar to go 1 month forward
calendar.StartDate = new DateTime(StartDate.Year, StartDate.Month, 1).AddMonths(1);
```

**Comment by PlusInf0Iulia** [comment]
when i try to change calendar.WeekdaysTextColor o week days font size nothing happens. is it a bug?

**Comment by rebeccaXam** [comment]
Hey PlusInf0Iulia,
yes you are right. The default colors where not set at creation, my bad. 
I will fix it in 1.0.6

**Comment by rebeccaXam** [comment]
Should be fixed now in 1.0.6
